src/modules/iotjs_module_pipe.c

```diff
@@ -163,23 +163,41 @@ JS_FUNCTION(PipeReadStart) {
 //   printf("write done with status: %d\n", status);
 // }
 
+static void write_utf8_string_cb(uv_write_t* req, int status) {
+  jerry_value_t callback = (jerry_value_t)(uintptr_t)req->data;
+  iotjs_jargs_t jargs = iotjs_jargs_create(1);
+  if (status) {
+    const char* uv_err = uv_strerror(status);
+    char err[128];
+    sprintf(err, "writeUtf8String error %s", uv_err);
```

Comment:
It would be better to remove the word `error` because the expected error is:

```
Error: writeUtf8String error .......
```

That's duplicated.

Reply:
Use `uv_strerror(status)` here directly?